examples/example_mountain_car.py

```diff
@@ -29,6 +29,26 @@
 
 import cgp
 
+# %%
+# Optionally, we can define the maximum number of generations of the
+# evolution on the command line and switch off the visualization of
+# the final champion (by running `python example_mountain_car.py
+# --no-visualize-final_champion`.
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--max-generations", type=int, help="max. number of generations in the evolution", default=1500
+)
+visualize_parser = parser.add_mutually_exclusive_group(required=False)
+visualize_parser.add_argument(
+    "--visualize-final-champion", dest="visualize_final_champion", action="store_true"
+)
+visualize_parser.add_argument(
```

Comment:
i don't understand why you need this option. isn't it the case that if `--visualize-final-champion` is provided it's true, if it's not provided it's false?

Reply:
solved with replacing by `docopt`